unreleased.md

```diff
@@ -1 +1,6 @@
 # Unreleased
+
+**Fixed:**
+
+- react-native-bpk-component-badge:
+  - Adjusted height to 20pt/dp for Android and IOS.
```

Comment:
`IOS` should have a lowercase `i` 😇 (iOS).

Reply:
My brain has been ignoring this for my entire life. :mind-blown: